python_modules/airline-demo/airline_demo/solids.py

```diff
@@ -114,6 +131,20 @@ def thunk(info):
     return info.config
 
 
+@solid(
+    name='thunk_database_engine',
+    outputs=[OutputDefinition(SqlAlchemyEngineType, description='The db resource.')]
+)
+def thunk_database_engine(info):
```

Comment:
very clever! didn't think these were serializable!

Reply:
too clever by half; in fact, they are not -- i am now doing this: https://github.com/dagster-io/dagster/blob/airline-demo-sql/python_modules/airline-demo/airline_demo/pipelines.py#L389